### PR TITLE
[ZH] Add `Gameplay/Ranked score` and `Gameplay/Total score`

### PR DESCRIPTION
**Pull request by Proxyerium**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

files changed:
- `Gameplay/Score/Ranked_score/zh.md`
- `Gameplay/Score/Total_score/zh.md`
- `Gameplay/Score/Total_score/en.md`

### CODE REVIEW

**Comment by Proxyerium** [comment]
动了一下英文，这种程度的更改要不要加`outdated`？
I had maden a small change in `Total_score/en.md`, is this change enough to add a `outdated` tag?

**Comment by CloneWith** [comment]
翻译的过期，要用“outdated_translation”吧？`outdated`标签用于英文原文内容 **不合事实** 的情况。

见 [ASC](https://osu.ppy.sh/wiki/Article_styling_criteria/Formatting)

This is to mark **translations** as outdated, not the content of original article, so it should use `outdated_translation` here. Please fix it:

```md
---
outdated_since: 3ffaca2ec4ce152fd0ec1f4d7f2dd8fdb99a2d8f
# should be this commit
outdated_translation: true
---
```

File: wiki/Gameplay/Score/Ranked_score/zh.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+*对于更多义项，参见：[Score (消歧义)](/wiki/Disambiguation/Score)。*
+
+**进榜总分**是玩家在线游玩过的所有上架（ranked）、社区喜爱（loved）和达标（approved）[难度](/wiki/Beatmap/Difficulty)所得到的最高分总和。
+
+## 历史
+
+[ppv1](/wiki/Performance_points/ppv1) 和 [ppv2](/wiki/Performance_points/ppv2) 出现之前，玩家的[排名](/wiki/Ranking)直接由进榜总分决定，这就导致了榜一经常换人，毕竟那个时候的 osu! 社区很小很小，谱面也寥寥无几。ppv1 于2012年7月24日完全替代进榜总分，随后于2014年1月27日被 ppv2 替代。
+
+谱面的分数如果超出旧[上架标准](/wiki/Ranking_criteria)规定的分数长度和上限，会被移至[达标谱面分类](/wiki/Beatmap/Category#approved)。这类谱面不给进榜总分，不过[总分](/wiki/Gameplay/Score/Total_score)和[表现分（PP）](/wiki/Performance_points)还是给的。
```
Comment (CloneWith) [inline wiki/Gameplay/Score/Ranked_score/zh.md:11]:
```suggestion
谱面的分数如果超出旧[上架标准](/wiki/Ranking_criteria)规定的长度和分数上限，会被移至[达标谱面分类](/wiki/Beatmap/Category#approved)。这类谱面不给进榜总分，不过[总分](/wiki/Gameplay/Score/Total_score)和[表现分（PP）](/wiki/Performance_points)还是给的。
```

Comment (CloneWith) [reply wiki/Gameplay/Score/Ranked_score/zh.md:11]:
```md
这类谱面不给进榜总分，不过[总分](/wiki/Gameplay/Score/Total_score)和[表现分（PP）](/wiki/Performance_points)还是给的。
```

可以考虑拓展一下“给分”：
`玩这类谱面不会增加进榜总分，但仍会获得总分与表现分。`

File: wiki/Gameplay/Score/Total_score/zh.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# 总分
+
+*对于更多义项，参见：[Score (消歧义)](/wiki/Disambiguation/Score)。*
+
+**总分**是玩家游玩任何谱面所得到的总分数值，也包括失败的谱面。
```
Comment (CloneWith) [inline wiki/Gameplay/Score/Total_score/zh.md:5]:
```suggestion
**总分**是玩家游玩所有谱面所得到的分数总和，也包括游玩失败的分数。
```

File: wiki/Gameplay/Score/Total_score/zh.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+玩家首次游玩一首上架谱面并取得成绩，总分和[进榜总分](/wiki/Gameplay/Score/Ranked_score)都会相应增加。往後的成绩如果没有超过最高记录，则只有总分会增加。
```
Comment (CloneWith) [inline wiki/Gameplay/Score/Total_score/zh.md:7]:
```suggestion
玩家首次游玩一张上架谱面并取得成绩，总分和[进榜总分](/wiki/Gameplay/Score/Ranked_score)都会相应增加。如果往后的成绩没有超过最高记录，则只会增加总分。
```
- “一首谱面”搭配不合适，可改成“一个”“一张”
- “往後”是繁体写法

File: wiki/Gameplay/Score/Total_score/zh.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+玩家的等级只由**总分**决定。升一级所需的分数由下面的公式算出：
+
+```
+score(n) = 5,000 / 3 * (4n^3 - 3n^2 - n) + 1.25 * 1.8^(n - 60) 如果 n <= 100
+score(n) = 26,931,190,827 + 99,999,999,999 * (n - 100)         如果 n > 100
+```
+
+介于游戏有四舍五入的机制，升到 100 级所需的总分数值和公式里的并不吻合。这是因为游戏会保留一个列表，里面的元素是预先计算好的每升一级所需的分数值，这些值都会约至整数。譬如，列表中第 81 个元素的值为 `score(81) - score(80) = 128,927,482.36216` ，约至整数得 128,927,482。在 100 级之前，总共有 +1.629 分的偏差，所以 `score(100) = 26,931,190,828.629` 和公式第二部分的 26,931,190,827 对不上。
+
+升到下一级的进度以长条样式显示在玩家的等级旁边。
```
Comment (CloneWith) [inline wiki/Gameplay/Score/Total_score/zh.md:20]:
```suggestion
升到下一级的进度以长条的形式显示在玩家的等级旁边。
```

File: wiki/Gameplay/Score/Total_score/zh.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+## 等级
+
+玩家的等级只由**总分**决定。升一级所需的分数由下面的公式算出：
+
+```
+score(n) = 5,000 / 3 * (4n^3 - 3n^2 - n) + 1.25 * 1.8^(n - 60) 如果 n <= 100
+score(n) = 26,931,190,827 + 99,999,999,999 * (n - 100)         如果 n > 100
+```
+
+介于游戏有四舍五入的机制，升到 100 级所需的总分数值和公式里的并不吻合。这是因为游戏会保留一个列表，里面的元素是预先计算好的每升一级所需的分数值，这些值都会约至整数。譬如，列表中第 81 个元素的值为 `score(81) - score(80) = 128,927,482.36216` ，约至整数得 128,927,482。在 100 级之前，总共有 +1.629 分的偏差，所以 `score(100) = 26,931,190,828.629` 和公式第二部分的 26,931,190,827 对不上。
```
Comment (CloneWith) [inline wiki/Gameplay/Score/Total_score/zh.md:18]: 我在考虑这里数字分隔方式的问题，需要改到以**万**分隔吗？例如 `128,927,482` -> `1,2892,7482`

Comment (Proxyerium) [reply wiki/Gameplay/Score/Total_score/zh.md:18]: 不必，三位分隔国际通用

File: wiki/Gameplay/Score/Total_score/zh.md (hunk 3 above)
Comment (Proxyerium) [reply wiki/Gameplay/Score/Total_score/zh.md:7]: 後 写惯了忘记改了🤪

**Review by CloneWith — APPROVED** [review]

File: wiki/Gameplay/Score/Ranked_score/zh.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# 进榜总分
+
+*对于更多义项，参见：[Score (消歧义)](/wiki/Disambiguation/Score)。*
+
+**进榜总分**是玩家在线游玩过的所有上架（ranked）、社区喜爱（loved）和达标（approved）[难度](/wiki/Beatmap/Difficulty)所得到的最高分总和。
+
+## 历史
+
+[ppv1](/wiki/Performance_points/ppv1) 和 [ppv2](/wiki/Performance_points/ppv2) 出现之前，玩家的[排名](/wiki/Ranking)直接由进榜总分决定，这就导致了榜一经常换人，毕竟那个时候的 osu! 社区很小很小，谱面也寥寥无几。ppv1 于2012年7月24日完全替代进榜总分，随后于2014年1月27日被 ppv2 替代。
```
Comment (YumeMuzi) [inline wiki/Gameplay/Score/Ranked_score/zh.md:9]: 日期罗马数字和中文之间加空格。

File: wiki/Gameplay/Score/Ranked_score/zh.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+谱面的分数如果超出旧[上架标准](/wiki/Ranking_criteria)规定的分数长度和上限，会被移至[达标谱面分类](/wiki/Beatmap/Category#approved)。游玩这类谱面不会增加进榜总分，不过仍能获得[总分](/wiki/Gameplay/Score/Total_score)和[表现分（PP）](/wiki/Performance_points)。
```
Comment (YumeMuzi) [inline wiki/Gameplay/Score/Ranked_score/zh.md:11]:
```suggestion
谱面的分数如果超出旧[上架标准](/wiki/Ranking_criteria)规定的分数长度和上限，会被移至[达标 (Approved) 谱面分类](/wiki/Beatmap/Category#approved)。游玩这类谱面不会增加进榜总分，但成绩仍能计入[总分](/wiki/Gameplay/Score/Total_score)和获得[表现分 (PP)](/wiki/Performance_points)。
```

**Review by TicClick — APPROVED** [review]

**Pull request merged**